src/proxy/proxy.cpp

```diff
@@ -169,6 +169,7 @@ void Proxy::slot_start()
             &m_groupManager,
             &Mmapper2Group::slot_parseGmcpInput);
     connect(mudTelnet, &MudTelnet::sig_relayGmcp, m_parserXml, &MumeXmlParser::slot_parseGmcpInput);
+    connect(mudTelnet, &MudTelnet::sig_relayGmcp, &m_mumeClock, &MumeClock::slot_parseGmcpInput);
```

Comment:
At some point I would personally suggest that we extract MumeClock from the proxy wiring and have it live outside, hooking up only to GameObserver. But not a blocker here

Reply:
Yeah, I wasn't sure on the best pattern here. I will pass the observer in and then connect it in the clock like what you did with the adventure panel. It is probably better for testing anyway since it's more isolated.